env_characterization/PathNode start
env_characterization/PathNode goal
---
env_characterization/PathNodeArray path